src/components/searchBar/searchBar.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import './searchBar.css';
+
+export default function SearchBar () {
+    return (
+        <header>
+            <div className="searchBarContainer">
+                <h4 className="searchBarContainer_title">Find your movies</h4>
+                <input className="searchBarContainer_input" placeholder="Enter movie title or genre"/>
+                <div className="searchBarContainer_underInput">
```

Comment:
why searchBar in header tag? It has to be single component inside component Header

Reply:
carelessness, forgot to remove before push, I will fix it